tests/test_inference.py

```diff
@@ -568,6 +568,72 @@ def verify_data_round_trip(
                 )
 
 
+class TestTruncateAncestorsRoundTripFromDisk(TestRoundTrip):
+    """
+    Tests that we can correctly round trip data when we truncate ancestral haplotypes
+    which have come from disk
+    """
+
+    def verify_data_round_trip(
+        self,
+        genotypes,
+        positions,
+        alleles=None,
+        sequence_length=None,
+        site_times=None,
+        individual_times=None,
+        ancestral_alleles=None,
+    ):
+        sample_data = self.create_sample_data(
+            genotypes,
+            positions,
+            alleles,
+            sequence_length,
+            site_times,
+            individual_times,
+            ancestral_alleles,
+        )
+        with tempfile.TemporaryDirectory() as d:
+            tsinfer.generate_ancestors(sample_data, path=d + "ancestors.tsi")
+            ancestors = tsinfer.AncestorData.load(d + "ancestors.tsi")
+            time = np.sort(ancestors.ancestors_time[:])
+            if (
```

Comment:
Maybe put the comment on a different line? Seems like unnecessary line breakage here